android/modules/ui/src/java/ti/modules/titanium/ui/WebViewProxy.java

```diff
@@ -127,6 +134,11 @@ public TiUIView createView(Activity activity)
 			postCreateMessage = null;
 		}
 
+		// Add script message handlers if attempted before the web-view could be created.
+		if (jsInterface != null) {
+			jsInterface.initializeJSInterface(webView);
+		}
```

Comment:
maybe add an error log in an `else` case? That way you'll get noticed if you call it too early (e.g. I still have to do it in the `open` event.

Reply:
you are right, sorry about that. Working fine now.